logging/src/main/java/org/wildfly/swarm/logging/LoggingFraction.java

```diff
@@ -51,6 +60,9 @@
 
     public static final String COLOR_PATTERN = "COLOR_PATTERN";
 
+    public static LoggingFraction loggingFraction;
```

Comment:
How so?

Ideally we want the methods of LoggingFraction to be chainable on an instance, but we don't need to hold an instance to make that happen.

Maybe I'm not understanding how this is meant to work